README.md

```diff
@@ -101,6 +93,64 @@ file.
 
 Note that new entry points will need the project to be unmounted and mounted again to be applied.
 
+### On the target machine (Wazo)
+
+#### Install dependencies
+Install rsync, and create a folder where repos must be available
+
+```sh
+apt update
+apt install rsync
+mkdir /usr/src/wazo  # or whatever your <local_source>
+```
+
+Install ssh
+```sh
+apt update
+apt install openssh-server
+```
+
+confiure ssh to enable root login
+```sh
+nano /etc/ssh/sshd_config
+```
+
+Add this line
+```sh
+PermitRootLogin yes
+```
+
+Restart sshd
+```sh
+sudo systemctl restart ssh
+```
+
+#### Cloning repos
+The repos you want to mount must be available in the local source folder
+
+```sh
+cd /usr/src/wazo
+git clone https://github.com/wazo-platform/the-repo-you-want
+```
+
+## Set the configuration file
+On your computer, edit the config file
+
+```sh
+nano ~/.config/wdk/config.yml
+```
+
+This is an example
+```yml
+# The wazo on wich you wish to work: [user@]wazo.example.com
+hostname: root@your-ip-or-hostname
+# The location of your local copy of the wazo code
+# You will clone git here
+local_source: ~/wazo
+# The location of the source code that is synchronized on the wazo server
+remote_source: /usr/src/wazo 
+```
+
```

Comment:
There's a `## Configuration` section already that instructs to create `~/.config/wdk/config.yml` based on `config.yml.sample`.
The config sample is self-documenting. 
Maybe we could add mentions in that section that `hostname` , `github_username` and `github_token` need to be changed.